rules/linux/credential_access_kubernetes_service_account_secret_access.toml

```diff
@@ -49,6 +49,7 @@ severity = "medium"
 tags = [
     "Domain: Endpoint",
     "Domain: Container",
+    "Domain: Kubernetes",
```

Comment:
"I think" K8s would be considered cloud workloads, but no objection to having it be a root domain.